.sass-lint.yml

```diff
@@ -0,0 +1,130 @@
+options:
+  merge-default-rules: false
+
+files:
+  include: 'indico/htdocs/sass/**/*.scss'
+  ignore: 'indico/htdocs/sass/lib/**/*'
+
+rules:
+  attribute-quotes: 1
+  bem-depth: 0
+  border-zero:
+    - 1
+    - convention: none
+  brace-style:
+    - 2
+    - style: 1tbs
+      allow-single-line: false
+  class-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: false
+  clean-import-paths: 2
+  declarations-before-nesting: 1
+  empty-args:
+    - 1
+    - include: true
+  empty-line-between-blocks: 1
+  extends-before-declarations: 2
+  extends-before-mixins: 1
+  final-newline: 2
+  force-attribute-nesting: 0
+  force-element-nesting: 0
+  force-pseudo-nesting: 0
+  function-name-format:
+    - 2
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: true
+  hex-length:
+    - 1
+    - style: short
+  hex-notation:
+    - 1
+    - style: lowercase
+  id-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: false
+  indentation:
+    - 2
+    - size: 4
+  leading-zero:
+    - 2
+    - include: true
+  max-file-line-count:
+    - 1
+    - length: 750
+  max-line-length:
+    - 2
+    - length: 120
+  mixin-name-format:
+    - 2
+    - convention: hyphenatedlowercase
+      allow-leading-underscore: true
+  mixins-before-declarations: 0
+  nesting-depth:
+    - 1
+    - max-depth: 4
+  no-attribute-selectors: 0
+  no-color-hex: 0
+  no-color-keywords: 0
+  no-color-literals: 0
+  no-combinators: 0
+  no-css-comments: 0
+  no-debug: 1
+  no-disallowed-properties: 0
+  no-duplicate-properties:
+    - 1
+    - exclude: [background-image, display]
```

Comment:
This is why I added it:

https://github.com/indico/indico/blob/master/indico/htdocs/sass/partials/_badges.scss#L132-L136

```css
//TODO replace with flexbox mixin once Compass is updated
display: block;
text-align: center;
display: -webkit-inline-flex;
display: inline-flex;
```